src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -173,31 +173,41 @@
                         "In this format the output will include the LOD score and also tumor-aware LOD score which can " +
                         "help assess identity even in the presence of a severe loss of heterozygosity with high purity (which could cause it to " +
                         "otherwise fail to notice that samples are from the same individual.) " +
-                        "A matrix output is also available to facilitate visual inspection of crosscheck results.\n " +
-                        "\n" +
+                        "A matrix output is also available to facilitate visual inspection of crosscheck results." +
+                        "<br /><br /> " +
```

Comment:
This is because the [documentation](https://gatk.broadinstitute.org/hc/en-us/articles/5358894366363-CrosscheckFingerprints-Picard-) currently outputs the top as a big block of text. I looked at another tool documentation as a template and added the html line breaks to (hopefully) get better formatting when it gets pulled onto the web.